app/api/server/v1/users.js

```diff
@@ -132,6 +133,36 @@ API.v1.addRoute('users.setActiveStatus', { authRequired: true }, {
 	},
 });
 
+API.v1.addRoute('users.deactivateIdle', { authRequired: true }, {
+	post() {
+		check(this.bodyParams, {
+			daysIdle: Match.Integer,
+			role: Match.Optional(String),
```

Comment:
You pass role, but below you don't seem to use it

Reply:
🙈 Fixed.